src/styles/blocks/card.css

```diff
@@ -0,0 +1,96 @@
+.card {
+  width: 200px;
+  box-sizing: border-box;
+
+  padding: 32px 16px 16px;
+
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  font-family: Roboto, Helvetica, sans-serif;
```

Comment:
font-family can be placed into reset.css

Reply:
Nope, move it normalize under `html` selector